ui/StatusQ/src/StatusQ/Components/StatusQrCodeScanner.qml

```diff
@@ -81,91 +152,146 @@ Item {
             maskSource: mask
         }
 
+        Loader {
+            active: root.highlightContentZone
+            sourceComponent: Rectangle {
+                color: "blue"
```

Comment:
"blue", not some color from `Theme.palette`?

Reply:
Well this one is just highlighting for testing purposes. Which is not used in the app actually.
I don't think it's needed to use any theme colors here 🙂